README.md

```diff
@@ -1,264 +1,505 @@
+
+Using Webpack to **[bundle](https://webpack.js.org/guides/getting-started/#creating-a-bundle)** this dependency chain up into as few output files as possible to the `dist/` directory means we have a `source/` (and `apps/`) folder that is structured the way we want to work, with a consistent output.
+
+Consider this dependency chain for the `apps/pl` app:
+
+                                                         <- @base
+                                                         <- jquery
+                                                         <- bootstrap/src/js/buttons
+                                    <- @atoms/button     <- _button.scss
+                                                         <- @base
+                                                         <- jquery
+                                                         <- bootstrap/src/js/cards
+    apps/pl <- source/design-system <- @molecules/card   <- _card.scss
+                                                         <- @base
+                                                         <- bootstrap/src/js/jumbotron
+                                    <- @organisms/header <- _header.scss
+
+### Font Icons
+
+Useful for small, frequently used icons that are a single color which is changeable via CSS.
+
+1. Place `filename.svg` in `source/_patterns/01-atoms/icon/svg/`
+1. Start up active server with `npm start` or compile via `npm run compile:pl|drupal`
+1. View new font icon demo page in Pattern Lab at [Atoms > Icon > Icons](http://localhost:8080/pl/?p=atoms-icons)
+1. Use either way:
+    - HTML class: `icon--filename`
+    - Sass Mixin: `@include icon(filename)`
+
+> IMPORTANT: Font icons are only compiled at the start of a webpack build. The webpack dev server will have to be restarted to see new icons appear in the font.
+
+#### Inline SVG
+
+Useful for larger, less frequently used vector images that potentially could be multi-color or able to animate.
 
-From your Drupal Twig templates in `templates/` you can `{% include %}`, `{% extends %}`, and `{% embed %}` your Pattern Lab Twig template files. Each of the top level folders has a Twig Namespace like `@organisms` and then you append the path down to the file like below.
+1. Place `file.svg` within a namespaced folder, like `source/_patterns/01-atoms/icon/svg/`.
+1. Use the special `_svg.twig` pattern to inline it completely. For instance, using the path in step 1, include it like so:
+    ```twig
+    {% include '@atoms/image/_svg.twig' with {
+      svgpath: '@atoms/icon/svg/file.svg',
+    } %}
+    ```
+1. OR just use the [`source`](https://twig.symfony.com/doc/2.x/functions/source.html) function provided by Twig: `{{ source('@atoms/icon/svg/file.svg') }}`
+
+### Static images
+
+Static image notes here.
```

Comment:
Hopefully something comes here.

Reply:
I want to get a bit more static image examples in so I can write up the design system + PL approach.